src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -216,23 +218,27 @@ export default function TokenDetails({
           <TokenDetailsSkeleton />
         )}
 
-        <RightPanel>
-          <Widget
-            token={token ?? undefined}
-            defaultField={Field.OUTPUT}
-            onTokenChange={navigateToWidgetSelectedToken}
-            onReviewSwapClick={onReviewSwapClick}
-          />
+        <RightPanel onClick={() => isBlockedToken && setOpenTokenSafetyModal(true)}>
```

Comment:
```suggestion
        <RightPanel onClick={isBlockedToken ? () => setOpenTokenSafetyModal(true) : null}>
```

? so we don't have an extra callback running unless needed?

Reply:
niceeee catch ty

Reply:
oops seems to lead to error actually / won't build